kinto_http/tests/functional.py

```diff
@@ -94,11 +94,6 @@ def test_buckets_deletion(self):
         assert buckets[0]['id'] == 'mozilla'
         self.assertRaises(BucketNotFound, self.client.get_bucket, 'mozilla')
 
-    def test_buckets_deletion_if_exists(self):
-        self.client.create_bucket('mozilla')
-        self.client.delete_buckets()
-        self.client.delete_buckets(if_exists=True)
```

Comment:
It has been done here: https://github.com/Kinto/kinto-http.py/pull/98/files#diff-9e98cb9d60ac0985ecb8fbfcca4148e8R97